lib/strings.js

```diff
@@ -230,55 +216,51 @@ const arrayRegExp = (
   return new RegExp('^' + ex + '$');
 };
 
-const section = (
-  // Split string by the first occurrence of separator
-  s, // string
-  separator // string, or char
-  // Returns: array of strings
-  // Example: rsection('All you need is JavaScript', 'is')
-  // Result: ['All you need ', ' JavaScript']
-) => {
+// Split string by the first occurrence of separator
+//   s - <string>
+//   separator - <string>, or char
+// Returns: <string[]>
+// Example: rsection('All you need is JavaScript', 'is')
+// Result: ['All you need ', ' JavaScript']
```

Comment:
I don't consider this a good idea to have special `Result` for `Example`, perhaps we could put it this way:
```
// Example: rsection('All you need is JavaScript', 'is')
//          -> ['All you need ', ' JavaScript']
```
OR
```
// Example: rsection('All you need is JavaScript', 'is') -> ['All you need ', ' JavaScript']
```
?